telegram/ext/jobqueue.py

```diff
@@ -126,7 +156,9 @@ def tick(self):
                 self.logger.debug('Running job %s', job.name)
```

Comment:
This debug would wrongly say that it's running a job even though it's not, if it's on an excluded day.